### PR TITLE
More shorthand alias added in Kubctl cheatsheet 

### PR DESCRIPTION
**Pull request by shekhar-rajak**
Fixes: https://github.com/kubernetes/kubernetes/issues/92635

Added generic shorthand alias for  frequently used commands.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Hi @shekhar-rajak

I'm not sure this change is the right fit for a section headed “Kubectl autocomplete”.

If readers wish to configure aliases in their shell, that's fine. I don't think it needs to be part of the documentation for Kubernetes though.

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -32,6 +32,28 @@ alias k=kubectl
...
+alias kgn="k get nodes -o wide"
+alias kgd="k get deployment -o wide"
+alias kgs="k get svc -o wide"
+
+
+# Describe K8S resources
+alias kdp="k describe pod"
+alias kdd="k describe deployment"
+alias kds="k describe service"
+
```
Comment (tengqm) [inline content/en/docs/reference/kubectl/cheatsheet.md:51]: Agree with @sftim, these alias are not related to autocompletion

**Comment by shekhar-rajak** [comment]
Thanks @tengqm @sftim for the review, any suggestions to correct the PR and close the kubernetes/kubernetes#92635

**Comment by tengqm** [comment]
Is alias a bash specific feature? Maybe add a separate subsection for bash alias, with one or two examples?

**Comment by sftim** [comment]
How about a blog article: “kubectl done quick” perhaps as a title.

The article could explain to readers how they can save time when interacting with their cluster using tools such as `kubectl`, `kubeadm`, maybe others too. I posted an [article](https://www.scalefactory.com/blog/2019/10/16/using-multiple-kubectl-versions-with-minimal-faff/) along broadly similar lines to my workplace blog about a year ago, that might give you an idea about what it could look like @shekhar-rajak

**Comment by shekhar-rajak** [comment]
Thanks @sftim @tengqm for your suggestions. We can see there is one alias already there : https://github.com/kubernetes/website/pull/25077/files#diff-9b5338cfe4b946f1b17ff38106ad8410dc6687e17e8327d3fc4864a5ee8a48c5R30 

```
You can also use a shorthand alias for `kubectl` that also works with completion:

alias k=kubectl
complete -F __start_kubectl k

```

I think we can have a separate section in the same page and out the bash alias related notes there. Also having blog is also a good idea. So should I move the above `alias k=kubectl` related notes in the separate section ? and work on blog post ?

**Comment by tengqm** [comment]
@shekhar-rajak I'll go with @sftim's suggestion. The `alias` concept is not a Kubernetes specific topic to document/host here. Writing a blog is an option. It is up to you. Another option could be add a subsection reminding users that creating alias can help save a lot time, with a link to an external source, e.g. https://en.wikipedia.org/wiki/Alias_(command).

**Comment by shekhar-rajak** [comment]
Yeah, I think creating subsection and inclusing few links/blog link should be ok.

**Comment by kbhawkey** [comment]
Hi @shekhar-rajak .
Do you need more information to move forward?

**Comment by kbhawkey** [comment]
Hi @shekhar-rajak .
I am closing this pull request. If you want to continue to make changes, you can reopen this PR.
Reach out if you need help or require more information. Thanks!
/close
